projects/aca-content/src/lib/components/search/search-input/search-input.component.html

```diff
@@ -36,8 +36,21 @@
         (submit)="onSearchSubmit($event)"
         (searchChange)="onSearchChange($event)"
       />
-      <mat-hint *ngIf="hasLibrariesConstraint" class="app-search-hint">{{ 'SEARCH.INPUT.HINT' | translate }}</mat-hint>
-
+      <mat-error *ngIf="searchInputControl.searchFieldFormControl.touched" class="app-search-error">
```

Comment:
If you will follow with that approach which I described here: 
https://github.com/Alfresco/alfresco-content-app/pull/4683/files#r2227689989

then it's possible that you will be able to move this condition for touched also to ngOnInit to subscribe. Not sure if you will be able to do it in statusChanges but if not then instead of statusChanges use events field (so this.searchFieldFormControl.events.pipe......). 

It can be implemented like: emit errors from search input control only when input is touched.

Please check if something like that will be working